heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
@kramasamy is this change required topology configs? I see many entries in this file that are not referenced in the codebase so wondering what it's primary function is.